src/framework/ui/button/button.component.tsx

```diff
@@ -50,7 +50,8 @@ export type ButtonElement = React.ReactElement<ButtonProps>;
  * Default is `false`.
  *
  * @property {string} status - Determines the status of the component.
- * Can be `primary`, `success`, `info`, `warning`, `danger` or `white`.
+ * Can be `primary`, `success`, `info`, `warning` or `danger`.
```

Comment:
`primary`, `success`, `info`, `warning`, `danger` or `control`

Reply:
Just not sure whether we will be able to include *control* status into the next release.

Reply:
Let's wait then